models/points.go

```diff
@@ -1911,8 +1941,8 @@ func (a Tags) String() string {
 // for data structures or delimiters for example.
 func (a Tags) Size() int {
 	var total int
-	for _, t := range a {
-		total += t.Size()
+	for i := range a {
```

Comment:
Does this avoid a copy of each item?

Reply:
Yes, something I noticed and shared in chat over the weekend. 

See this https://godbolt.org/g/dtR9XM

`SeriesKeyBytesSizeA` main loop to sum the lengths is really efficient:

```asm
  leaq (BX)(BX*2), DI
  shlq $4, DI
  movq 8(DX)(DI*1), R8
  addq R8, AX
  leaq 32(DX)(DI*1), DI
  addq (DI), AX
  movq SI, BX
  cmpq BX, CX
  jlt 23
 ```

`SeriesKeyBytesSizeB` copies the tag:

```asm
  leaq ""..autotmp_15+48(SP), DI
  movq DX, SI
  duffcopy $854
  leaq "".tag(SP), DI
  leaq ""..autotmp_15+48(SP), SI
  duffcopy $854
  addq $48, DX
  incq BX
  movq "".tag+8(SP), R8
  addq R8, AX
  leaq "".tag+32(SP), R8
  addq (R8), AX
  cmpq BX, CX
  jlt 43
```